common/params_pyx_setup.py

```diff
@@ -0,0 +1,21 @@
+import os
+from distutils.core import Extension, setup
+from Cython.Build import cythonize
+from common.cython_hacks import BuildExtWithoutPlatformSuffix
+from common.basedir import BASEDIR
+
+sourcefiles = ['params_wrapper.pyx']
+extra_compile_args = ["-std=c++17"]
```

Comment:
why is this 17?

Reply:
Was done in the original params PR. Will change it what we use everywhere else. We should switch to 17 at some point though, it has some nicer for loops over maps.